docs/installation.md

```diff
@@ -59,39 +58,6 @@ If you want to build your own kraken image see [here](https://github.com/redhat-
 Refer [Instructions](https://github.com/redhat-chaos/krkn/blob/main/containers/README.md) on how to deploy and run Kraken as a Kubernetes/OpenShift deployment.
 
 
```

Comment:
Maybe let's add a reference to the helm chart and tekton way of running things in this document but say it was created by a specific user so any issues with it need to be reported to the upstream repo

Reply:
Think it might be better to just have pointers in the guide to rule out any confusions given that the tekton way of running things is not natively maintained in Kraken. Thoughts?

Reply:
Maybe lets keep a reference to using this type of installation type but give the same notice as the other spot in the index doc. Again think a separate file all together might be good for this info though